claimer/src/check-outstading-debts.ts

```diff
@@ -0,0 +1,40 @@
+import { gql } from 'graphql-request'
+import { logger } from './logger'
+
+const GET_OUTSTANDING_DEBTS = gql`
+  query {
+    relayPools {
+      items {
+        chainId
+        contractAddress
+        outstandingDebt
+        origins {
```

Comment:
add a flag to track `disabled` origin or check here if `maxDebt != 0` to make sure we ignore disabled origin